src/sentry/conf/server.py

```diff
@@ -1133,13 +1133,23 @@ def get_raven_config():
 
 # TODO(dcramer): move this to sentry.io so it can be automated
 SDK_VERSIONS = {
-    'raven-js': '3.9.1',
-    'raven-python': '5.32.0',
-    'sentry-laravel': '0.5.0',
-    'sentry-php': '1.6.0',
+    'raven-java': '7.8.3',
+    'raven-java:android': '7.8.3',
```

Comment:
These are all from the same package, right? If so, I wonder if we can support this pattern more explicitly, and assume `raven-java:<thing>` is all the same?